cloud/amazon/ec2_vpc_peering.py

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/python
+#
+DOCUMENTATION = """
+---
+module: ec2_vpc_peer
+short_description: create or remove a peering connection between to ec2 VPCs.
+description:
+    -
+options:
+  vpc_id:
+    description:
+      - VPC id of the requesting VPC.
+    required: true
+  vpc_peer_id:
+    description:
+      - VPC id of the accepting VPC.
+    required: true
+  state:
+    description:
+      - Create or delete the peering connection.
+    required: false
+    default: present
+    choices: ['present', 'absent']
+  wait_timeout:
+    description:
+      - How long to wait for peering connection state changes, in seconds
+    required: false
+    default: 10
+  update_routes:
+    description:
+      - Whether to update the VPC route tables to add the peering connection.
+    required: false
+    default: true
+  region:
+    description:
+      - The AWS region to use.  Must be specified if ec2_url is not used. If not specified then the value of the EC2_REGION environment variable, if any, is used.
+    required: false
+    default: null
+    aliases: ['aws_region', 'ec2_region']
+  aws_secret_key:
+    description:
+      - AWS secret key. If not set then the value of the AWS_SECRET_KEY environment variable is used.
+    required: false
+    default: None
+    aliases: ['ec2_secret_key', 'secret_key']
+  aws_access_key:
+    description:
+      - AWS access key. If not set then the value of the AWS_ACCESS_KEY environment variable is used.
+    required: false
+    default: None
+    aliases: ['ec2_access_key', 'access_key']
+
+requirements: [ "boto" ]
+"""
+
+import sys
+import time
+
+try:
+    import boto.ec2
+    import boto.vpc
+    import boto.exception
+except ImportError:
+    print "failed=True msg='boto required for this module'"
+    sys.exit(1)
```

Comment:
Because back in February this was the standard pattern used by the core modules. :-) 

Have updated it to use the same pattern that they now use.